pkg/action/list.go

```diff
@@ -322,3 +335,15 @@ func (l *List) SetStateMask() {
 
 	l.StateMask = state
 }
+
+// Create a clone of the config with a new namespace
+func (cfg *Configuration) CloneWithNewNamespace(namespace string) (*Configuration, error) {
+	newConf := &Configuration{}
+	newConf.Releases = cfg.Releases
+	getter := settings.RESTClientGetter()
```

Comment:
getter := cfg.RESTClientGetter.(genericclioptions.RESTClientGetter) does this type cast not work?